app/components/at_mention/at_mention.js

```diff
@@ -79,12 +79,12 @@ export default class AtMention extends React.PureComponent {
         }
     };
 
-    getUserDetailsFromMentionName(props) {
-        let mentionName = props.mentionName.toLowerCase();
+    getUserDetailsFromMentionName(name, usersByUsername = {}) {
```

Comment:
can you move this one to be an util function like you did in #2280, actually I think you can combine both functions as they do the same thing?

Reply:
Sure, I combined both.  I think there's no problem.  I tested it and looks good to me.